### PR TITLE
Fixed pylint(2.10.2) errors in AC files

### PR DESCRIPTION
**Pull request by Anna-Mironova**
This PR includes fixes for linter errors in accuracy checker files since upgrade to pylint version 2.10.2

### CODE REVIEW

**Review by eaidova — APPROVED** [review]

File: tools/accuracy_checker/data/test_models/pytorch_model/samplenet.py [hunk 1]
```diff
@@ -14,13 +14,13 @@
...
 
-import torch.nn as nn
+from torch import nn
 import torch.nn.functional as F
 
 
 class SampLeNet(nn.Module):
     def __init__(self):
-        super(SampLeNet, self).__init__()
+        super().__init__()
```
Comment (eaidova) [inline tools/accuracy_checker/data/test_models/pytorch_model/samplenet.py:23]: please return back, it is pytorch syntaxis
https://pytorch.org/docs/stable/generated/torch.nn.Module.html

Comment (Anna-Mironova) [reply tools/accuracy_checker/data/test_models/pytorch_model/samplenet.py]: Done
